### PR TITLE
virt-test: enforce_quit:Win8 guest lack of 'enforce' params to the cmd line

### PR DESCRIPTION
**Pull request by XueqiangWei**
Signed-off-by: Xueqiang Wei <xuwei@redhat.com>

### CODE REVIEW

**Comment by XueqiangWei** [comment]
ID: 1182917

**Comment by CongSmile** [comment]
Acked-by: Cong Li <coli@redhat.com>

File: shared/cfg/guest-os/Windows/Win8.cfg [hunk 1]
```diff
@@ -4,7 +4,7 @@
     shutdown_command = shutdown /s /f /t 0
     reboot_command = shutdown /r /f /t 0
     # it can be removed while bug 821741 is fixed
-    cpu_model_flags = ",+sep"
```
Comment (xutian) [inline shared/cfg/guest-os/Windows/Win8.cfg]: Please check 821741, do we still need this configuration now.

thanks,
@xutian

**Comment by lmr** [comment]
Hi @XueqiangWei. As you might know, we're moving virt-test to avocado-vt, so I please ask you to consider re-implementing this functionality in the later. The new repo is:

https://github.com/avocado-framework/avocado-vt

Please keep in mind the following guidelines when creating the new PR:

1) Most of the time you can pick the patches of your original PR and merge them on a new branch with minimal changes.
3) Don't use autotest APIs, since we're moving away from using them. Please review the code and change things like `utils.system` -> `process.system`, `utils.run` -> `process.run`, so on and so forth.

Thanks, and I'm sorry for any inconveniences we might have caused your team.

**Pull request closed without merging**